docs/advanced-topics/verification.rst

```diff
@@ -0,0 +1,126 @@
+Verifying a test parameter file
+===============================
+
+1.  Go to
+    https://dockstore.org/static/swagger-ui/index.html#/extendedGA4GH/toolsIdVersionsVersionIdTypeTestsPost
+2.  Click “Try it out”
+3.  Provide a “type”. See the
+    description for allowable values.
+4.  Provide the TRS ID for the tool/workflow being verified.
+    For example, the `dockstore-tool-md5sum`_ tool has the TRS ID:
+    “quay.io/briandoconnor/dockstore-tool-md5sum” as shown in the “Info”
+    tab with the label: “TRS CWL Path” or “TRS WDL Path”
```

Comment:
TRS Labels on info tab doesn't say 'Path'. 
![image](https://user-images.githubusercontent.com/23464754/64374847-80f67880-cfd9-11e9-8326-9208ccc074be.png)

Also swagger label just calls it "ID",  the label or description of the label should probably mention TRS. I don't have a particularly strong opinion on what the it should be called (ID vs TRS ID vs TRS Path), but the naming should be consistent between swagger, UI, and docs

Reply:
Can't really change much in my opinion.  The ID is implied to be TRS ID because the swagger.yaml is in the TRS repo.  The documentation would be more confusing if it didn't specify TRS ID because people would likely use the Dockstore Tool or Workflow ID. The UI doesn't explicitly have a field called TRS ID which is odd but :man_shrugging: